### PR TITLE
Save shortcut and cleanups in Stringdialog

### PR DESCRIPTION
**Pull request by oscargus**
Removed the button, but kept the shortcut plus all the cleanups from the previous PR.

### CODE REVIEW

**Comment by matthiasgeiger** [comment]
:+1: 

Just one thing: As @simonharrer already mentioned in the earlier pull request: It is not necessary to create all those serial versions UIDs :wink:

**Comment by oscargus** [comment]
Yeah, I noted that (and changed my warning preferences). Didn't get the idea that they should be removed though. :-) (But I will for sure not add any more.)

File: src/main/java/net/sf/jabref/gui/StringDialog.java [hunk 1]
```diff
@@ -133,11 +137,9 @@ protected boolean accept(Component c) {
...
         RemoveStringAction removeStringAction = new RemoveStringAction(this);
         am.put("remove", removeStringAction);
-        //im.put(prefs.getKey("String dialog, move string up"), "up");
-        //am.put("up", stringUpAction);
-        //im.put(prefs.getKey("String dialog, move string down"), "down");
-        //am.put("down", stringDownAction);
-        im.put(prefs.getKey("Close dialog"), "close");
+        im.put(prefs.getKey("Save database"), "save");
+        am.put("save", saveAction);
+       im.put(prefs.getKey("Close dialog"), "close");
```
Comment (simonharrer) [inline src/main/java/net/sf/jabref/gui/StringDialog.java]: wrong indent

**Comment by simonharrer** [comment]
Can you please remove the IDs and fix the indent the commented place? Then it is good to be merged. 

Thanks for the improvements!

**Comment by oscargus** [comment]
OK! I've changed that, ran the Eclipse formatter and added formatter:off/on for multi-localization commands.

**Pull request merged**